src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php

```diff
@@ -321,14 +321,14 @@ protected function seeJsonContains(array $data, $negate = false)
 
         $actual = json_encode(Arr::sortRecursive(
             (array) $this->decodeResponseJson()
-        ));
+        ), JSON_PRETTY_PRINT);
 
         foreach (Arr::sortRecursive($data) as $key => $value) {
             $expected = $this->formatToExpectedJson($key, $value);
 
             $this->{$method}(
                 Str::contains($actual, $expected),
-                ($negate ? 'Found unexpected' : 'Unable to find')." JSON fragment [{$expected}] within [{$actual}]."
+                ($negate ? 'Found unexpected' : 'Unable to find')." JSON fragment\n[{$expected}]\nwithin\n[{$actual}]."
```

Comment:
Probably use the system EOL constant here since we want readability in the test output.